backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/vdsbroker/VdsBrokerObjectsBuilder.java

```diff
@@ -1602,6 +1602,13 @@ private static void updateVDSDomainData(VDS vds, Map<String, Object> struct) {
                         acquired = (Boolean)internalValue.get(VdsProperties.acquired);
                     }
                     data.setAcquired(acquired);
+
+                    Boolean valid = Boolean.FALSE;
+                    if (internalValue.containsKey(VdsProperties.valid)) {
+                        valid = (Boolean)internalValue.get(VdsProperties.valid);
+                    }
```

Comment:
is it reported since 4.2?

Reply:
https://github.com/oVirt/vdsm/blame/3b87510e20a45a5d5d17cd57f5fe5d475ceb2179/lib/vdsm/api/vdsm-api.yml#L2152
last change was 6 years ago so should be good